@navikt/core/react/src/list/List.tsx

```diff
@@ -42,47 +45,34 @@ export const List = forwardRef<HTMLDivElement, ListProps>(
     },
     ref,
   ) => {
-    const ariaId = useId();
+    const { size: contextSize } = useContext(ListContext);
 
-    const { isNested, size: _size } = useContext(ListContext);
-
-    const listSize = size ?? _size ?? "medium";
+    const listSize = size ?? contextSize;
     return (
       <ListContext.Provider
         value={{
           listType: ListTag,
-          isNested: isNested === null ? false : true,
           size: listSize,
         }}
       >
-        <div
+        <BodyLong
```

Comment:
The reason for putting BodyLong here was to get it around the `::marker` (the numbers), so that I don't need CSS to set font-size etc. for that. I tried doing `<li as={BodyLong}` first, but got problems with TS. Then I thought I might as well put it here instead of on each item.